apps/firebase/rules.bolt

```diff
@@ -24,6 +24,7 @@ path /v3/config {
 // Don't allow reading /v3/channels, since that would expose the list of channels.
 path /v3/channels/{channelId} {
   read() { canAccessChannel(channelId) }
+  write() { canAccessChannel(channelId) }
```

Comment:
this line is concerning because it breaks the strategy outlined above for ensuring there are no unsecured nodes in our firebase:
```
// 1. a node with a write() rule must not have an ancestor with a write() rule.
```
Rather than changing firebase rules to allow this or rethinking this entire strategy, I think it will be easier to revert this change to the rules and just delete the nodes you need to:
```
/v3/channels/{channelId}/current_tables
/v3/channels/{channelId}/metadata
/v3/channels/{channelId}/storage
```
It seems ok not to erase `/v3/channels/{channelId}/counters`, which might be difficult given the rules on those.

In the future, if you ever want to delete an entire channel such as when we delete a user's account, you could do that from the server, which these security rules won't apply to.

Reply:
Unfortunately I think we need to erase `/v3/channels/{channelId}/counters`. That node is used as the source of truth for the list of tables in the project.
Without deleting `/counters`, if I make a project with a table `mytable`, then start over, I will still see `mytable` in the list

Reply:
can you use `/v3/channels/{channelId}/current_tables` as the source of truth for what tables exist instead?

it seems fine to erase `counters` if you need to -- I just wasn't quite sure if the existing security rules would let you.

If doing this on the client gets to be too complicated, one alternate approach is to send a request to the server and then make a REST API request to delete the entire channel via `firebase_helper.rb`.